docs/migration.md

```diff
@@ -0,0 +1,128 @@
+# Migrate from PostgreSQL Community to Percona Distribution for PostgreSQL 
```

Comment:
"PostgreSQL Community" somehow sounds weird to me (it reminds me of "Community Version" vs. "Enterprise Version"), but I admit I don't have a better suggestion. Maybe "Vanilla PostgreSQL" or "Upstream PostgreSQL"?

Reply:
It took me a while to figure put what Vanilla Postgres is. Perhaps we should use just PostgreSQL?

Reply:
That's fine, but then we should clarify what we mean by "PostgreSQL" in a sentence - maybe "PostgreSQL binaries provided by the PostgreSQL community"? "by the upstream community"?

Reply:
Our product name is Percona Distribution for PostgreSQL :)

Reply:
Fixed